src/test/java/uk/ac/ebi/eva/commons/models/converters/data/DBObjectToVariantConverterTest.java

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright 2015-2016 OpenCB
+ * Copyright 2017 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.ac.ebi.eva.commons.models.converters.data;
+
+import com.google.common.collect.Lists;
```

Comment:
Then please remove the dependency and change the Guava Lists.newArrayList("NA001", "NA002", "NA003"); for base java standard code.